static/src/javascripts/projects/admin/modules/abtests/abtest-report-item.js

```diff
@@ -10,7 +10,7 @@ import bonzo from 'bonzo';
 import debounce from 'lodash/functions/debounce';
 
 class ABTestReportItem extends Component {
-    constructor(config: any): void {
+    constructor(config: Object): void {
```

Comment:
Not strictly part of this es6 conversion but 'Object' is the correct type of config.